app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -238,10 +238,15 @@ open class DataStore(
     private fun sync() {
         resetSupport(support)
```

Comment:
Can we find a better name for this method to make it more clear?

Reply:
This should be addressed in #885

Reply:
Agreed.